bonfire/elastic_logging.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import requests
+from concurrent.futures import ThreadPoolExecutor
+
+import bonfire.config as conf
+
+
+class AsyncElasticsearchHandler(logging.Handler):
+    def __init__(self, es_url):
+        super().__init__()
+        self.es_url = es_url
+        self.executor = ThreadPoolExecutor(max_workers=10)
+
+    def emit(self, record):
+        log_entry = self.format(record)
+        if conf.ENABLE_TELEMETRY == 'true' and conf.BONFIRE_BOT:
```

Comment:
If we make it a bool, this line can be changed to:
```suggestion
        if conf.ENABLE_TELEMETRY:
```

If you are trying to use the bot check to determine whether telemetry is "on" or "off", honestly I think we can just use 1 env var here. We'll just set up the jenkins jobs to pass `ENABLE_TELEMTRY=true` as an env var and that can take care of it.